araproc/analysis/standard_reco.py

```diff
@@ -236,34 +243,81 @@ def do_standard_reco(self, waveform_bundle):
         # reco_results["pulser_h"] = {"corr" : corr_pulser_h, 
         #                             "theta" : theta_pulser_h,
         #                             "phi" : phi_pulser_h,
-        #                             "map" : pulser_map_h
+        #                             "map" : pulser_map_h,
+        #                             "radius" : self.calpulser_r_library[self.station_id]
         #                             }
 
-        # make a 300 m map in V
-        distant_map_v = self.rtc_wrapper.correlators["distant"].GetInterferometricMap(
+        # make a 300 m map in V (Direct rays)
+        distant_map_v_dir = self.rtc_wrapper.correlators["distant"].GetInterferometricMap(
             self.pairs_v,
             corr_functions_v,
             0
         )
-        corr_distant_v, phi_distant_v, theta_distant_v = mu.get_corr_map_peak(distant_map_v)
-        reco_results["distant_v"] = {"corr" : corr_distant_v, 
-                                    "theta" : theta_distant_v,
-                                    "phi" : phi_distant_v,
-                                    "map" : distant_map_v
-                                    }
 
-        # make a 300 m map in H
-        distant_map_h = self.rtc_wrapper.correlators["distant"].GetInterferometricMap(
+        # make a 300 m map in V (Refracted/Reflected rays)
+        distant_map_v_ref = self.rtc_wrapper.correlators["distant"].GetInterferometricMap(
+            self.pairs_v,
+            corr_functions_v,
+            1
+        )
+
+        # Get the correlation, phi, and theta for both maps
+        corr_distant_v_dir, phi_distant_v_dir, theta_distant_v_dir = mu.get_corr_map_peak(distant_map_v_dir)
+        corr_distant_v_ref, phi_distant_v_ref, theta_distant_v_ref = mu.get_corr_map_peak(distant_map_v_ref)
+
+        # Store the direct rays results
+        reco_results["distant_v_dir"] = {
+            "corr": corr_distant_v_dir, 
+            "theta": theta_distant_v_dir,
+            "phi": phi_distant_v_dir,
+            "map": distant_map_v_dir,
+            "radius": self.distant_events_r_library[1]
+        }
+
+        # Store the refracted/reflected rays results
+        reco_results["distant_v_ref"] = {
+            "corr": corr_distant_v_ref, 
+            "theta": theta_distant_v_ref,
+            "phi": phi_distant_v_ref,
+            "map": distant_map_v_ref,
+            "radius": self.distant_events_r_library[1]
```

Comment:
I would prefer if you did `self.rtc_wrapper.correlators["distant"].GetRadius()`

Reply:
That way the only thing that's important is that you have your handle on the right correlator. It's more modular (I think...)

Reply:
Ah, I didn't know I could get the radius from there. Thanks!

Reply:
Yup! :)

Reply:
Done. Should I merge or wait for @marcomuzio?

Reply:
Merge away!